README.MD

```diff
@@ -0,0 +1,24 @@
+# Vandelay
+![About Vandelay](images/About.png)
```

Comment:
Not sure the About window is that informative. How about just the main window shot between the next paragraph and the "Available units:"?
Also, I'd prefer to have the screenshot including the window border. The yellow tabs are a hallmark of Haiku and should be signaled whenever possible to create Haiku-awareness. :)